vcd/resource_vcd_vapp_vm_multinetwork_test.go

```diff
@@ -386,6 +448,25 @@ resource "vcd_vapp_vm" "{{.VMName}}" {
     is_primary         = false
 	mac                = "00:00:00:11:11:11"
   }
+
+  network {
+    type               = "vapp"
+    name               = vcd_vapp_network.vappRoutedNet.name
```

Comment:
Just in case: is it on purpose that this test doesn't have a vApp isolated network like the above one?

Reply:
update removes some of the nics